<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>711</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionBold"/>
    <addaction name="actionSuper_script"/>
    <addaction name="actionSubscript"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionWord_Wrap"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/Images/Images/folder.png</normaloff>:/Images/Images/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/Images/Images/diskette.png</normaloff>:/Images/Images/diskette.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/Images/Images/saveAs.png</normaloff>:/Images/Images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/Images/Images/remove.png</normaloff>:/Images/Images/remove.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/Images/Images/undo-circular-arrow.png</normaloff>:/Images/Images/undo-circular-arrow.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/Images/Images/cutting.png</normaloff>:/Images/Images/cutting.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/Images/Images/copy.svg</normaloff>:/Images/Images/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/Images/Images/paste.png</normaloff>:/Images/Images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/Images/Images/button.png</normaloff>:/Images/Images/button.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionWord_Wrap">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/Images/Images/icons8-word-wrap-50.png</normaloff>:/Images/Images/icons8-word-wrap-50.png</iconset>
   </property>
   <property name="text">
    <string>Word Wrap</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionSuper_script">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Superscript</string>
   </property>
  </action>
  <action name="actionSubscript">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Subscript</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="AppResources.qrc"/>
 </resources>
 <connections/>
</ui>
